local/test-binary

```diff
@@ -15,14 +15,26 @@ set -o errexit
 set -o xtrace
 
 WORKDIR_ABS=$(cd ${1:-./build}; pwd -P)
+rm -fr ${WORKDIR_ABS}/PS
```

Comment:
Why? Does this fix some other JEN?

Reply:
it fixes the following  issue: if test is running on the same node after build, then 3 mysqld binaries may be found in WORKDIR_ABS - from build tree, from installation directory and from tarball

Reply:
It would be more robust to delete all the non-needed copies then?
This would also help with find | head -1 below (both faster and removes the need for head -1).
Also how do we know that head -1 there picks the right copy?